File: CH_02_01.py
MY_NAME = "Ronnie"


def say_hi(name):
    return f"Hi {name} 👋!"


class Pet:
    def __init__(self, name, age):
        self.name = name
        self.age = age


mona = Pet("Mona", 6)

print(say_hi(MY_NAME))

File: CH_02_01.js
const myName = "Ronnie";

function sayHi(name) {
  return `Hi ${name} 👋!`;
}

class Pet {
  constructor(name, age) {
    this.name = name;
    this.age = age;
  }
}

const mona = new Pet("Mona", 6);

console.log(sayHi(myName));
